nextjs-app-v13/package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "nextjs-app",
+  "name": "nextjs-app-v13",
   "version": "0.1.0",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "preview": "next start",
     "lint": "next lint",
-    "playwright:show-report": "npx playwright show-report ../playwright-reports/nextjs-app-playwright-report",
-    "test:system": "APP_NAME=nextjs-app playwright test -c ../playwright.config.ts",
+    "playwright:show-report": "npx playwright show-report ../playwright-reports/nextjs-app-v13-playwright-report",
+    "test:system": "APP_NAME=nextjs-app-v13 playwright test -c ../playwright.config.ts",
     "test:visual": "PERCY_TOKEN=${PERCY_TOKEN_PIE_APERTURE_NEXT} npx percy exec -- wdio run ./wdio.conf.js",
```

Comment:
Is it worth renaming the percy token variable?

Reply:
This is to be addressed in a seperate ticket as changing it now will cause some issues with the existing tests.